### DISCUSSION TITLE
Enhancing pigment-css to be atomic

### DISCUSSION THREAD

**Discussion by paulm17** [mui/pigment-css#45] [Discussions]
Firstly, thank you for making pigment-css.  Ever since vercel announced RSC and the closure of runtime css-in-js.  I've been waiting for a library that I can work with.  I've tried kuma-ui, panda-css, linera, griffel, stylexjs, and many others. 

I've managed to get pigment-css to be atomic.  See screenshot below.

Many thanks!

![Image Description](https://i.imgur.com/wF6UXw2.png)

**Comment by Amir782Shahzad** [comment]
good

**Comment by Axe1200** [comment]
Super
